embed-api-v3.rst

```diff
@@ -0,0 +1,119 @@
+.. post:: December 2, 2021
+   :tags: api, feature, grant
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      We are thrilled to announce the availability of Read the Docs Embed API v3,
+      along with its official client, sphinx-hoverxref 1.0.
+      We invite the community to try them out and let us know their feedback.
+
+Announcing Embed API v3 and sphinx-hoverxref 1.0
+================================================
+
+We are thrilled to announce the availability of Read the Docs Embed API v3,
+along with its official client, sphinx-hoverxref 1.0.
+This work has been possible in part thanks to the
+:doc:`the CZI grant we received </czi-grant-announcement>`.
+
+Why using content embedding?
```

Comment:
```suggestion
Why use content embedding?
```

I think this is the proper grammar, but feels like a weird header for the opening of a blog post. Perhaps "The value of content embedding" or similar?

Reply:
Used your suggestion 👍🏽